Microsoft.Toolkit.Uwp.SampleApp/SamplePages/AdaptiveGridView/AdaptiveGridViewPage.xaml

```diff
@@ -26,16 +26,30 @@
         <Grid.RowDefinitions>
             <RowDefinition Height="Auto" />
             <RowDefinition Height="*" />
+            <RowDefinition Height="Auto" />
         </Grid.RowDefinitions>
+        <Grid.ColumnDefinitions>
+            <ColumnDefinition Width="Auto" />
+            <ColumnDefinition Width="*" />
+        </Grid.ColumnDefinitions>
 
         <TextBlock x:Name="SelectedItemCountTextBlock"
                    Margin="0,12"
+                   Grid.ColumnSpan="2"
                    HorizontalAlignment="Center"
                    FontFamily="Segoe UI Light"
                    FontSize="21"
                    Text="You haven't selected any items"
                    Visibility="{Binding IsItemClickEnabled.Value, Converter={StaticResource BoolToVisibilityConverter}}" />
 
-        <Grid x:Name="XamlRoot" Grid.Row="1" />
+        <Grid x:Name="XamlRoot" Grid.ColumnSpan="2" Grid.Row="1" />
+
+        <TextBlock FontFamily="Segoe UI Light"
+                   FontSize="21" 
+                   Text="Number of Photos" 
+                   Grid.Row="2" 
+                   Margin="5" />
+        
+        <Slider x:Name="NumberSlider" Grid.Row="2" Grid.Column="1" Margin="30,5" StepFrequency="1"   />
```

Comment:
I would use a margin of 32 and 4 here or 28 and 4, to work well with scaling

Reply:
No probs, will change and add to pull request